.env.development

```diff
@@ -5,6 +5,7 @@ APPOINTMENT_NOTIFICATION_HOUR_OF_DAY_START=14
 CALL_SESSION_REDIS_HOST=localhost
 CALL_SESSION_REDIS_POOL_SIZE=12
 CALL_SESSION_REDIS_TIMEOUT_SEC=1
+CUSTOM_DRUG_CATEGORY_ORDER=["hypertension_ccb", "hypertension_arb", "hypertension_diuretic"]
```

Comment:
We can also do `hypertension_ccb,hypertension_arb,hypertension_diuretic` and split(",") in code.

Reply:
Put this in `CountryConfig` instead